Migrations/migrations.sql

```
@@ -0,0 +1,243 @@
+    REFERENCES [dbo].[restaurants] ([id])
+GO
+
+-- Add CHECK constraints to orders
+IF NOT EXISTS (
+    SELECT * FROM sys.check_constraints WHERE name = 'CK_orders_mode_of_payment'
+)
+    ALTER TABLE [dbo].[orders]  WITH CHECK ADD CONSTRAINT [CK_orders_mode_of_payment] CHECK  (([mode_of_payment]='card' OR [mode_of_payment]='upi'))
+GO
+
+IF NOT EXISTS (
+    SELECT * FROM sys.check_constraints WHERE name = 'CK_orders_status'
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)